django/api/signals.py

```diff
@@ -29,7 +29,6 @@ def after_household_member_save(sender, instance, created, **kwargs):
         if settings.EMAIL["SEND_EMAIL"]:
             async_task(
                 "api.tasks.send_household_confirm",
-                instance.email,
```

Comment:
Wouldn't this break the household confirm email? Although I think in the meeting someone mentioned not sending the household confirm email to the spouse at all... so in this case, the args would be (application.email, application.id) ?

Reply:
Yeah I think this would break it too.

Reply:
woops thanks for catching that! I guess I hadn't seen the final confirmation message yet so didn't notice it not coming in! its fixed now in my last commit :)